### PR TITLE
Remove getrandom dependency being pulled

### PR DESCRIPTION
**Pull request by ueco-jb**
closes https://github.com/CosmWasm/cw-storage-plus/issues/19

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -36,7 +36,8 @@ thiserror = "1.0.13"
 
 [dev-dependencies]
 criterion = { version = "0.3", features = ["html_reports"] }
-rand = "0.8"
+rand = { version = "0.8", default-features = false }
+rand_xoshiro = { version = "0.6.0", default-features = false }
```
Comment (maurolacy) [inline Cargo.toml:40]: Is this because `rand_xoshiro` is `no_std`?

**Review by maurolacy — APPROVED** [review]
LGTM.

An alternative can as well be to remove those benchmarks altogether.

**Review by webmaster128 — APPROVED** [review]
Nice

File: Cargo.toml [hunk 2]
```diff
@@ -36,7 +36,8 @@ thiserror = "1.0.13"
 
 [dev-dependencies]
 criterion = { version = "0.3", features = ["html_reports"] }
-rand = "0.8"
+rand = { version = "0.8", default-features = false }
```
Comment (webmaster128) [inline Cargo.toml:39]:
```suggestion
# Deactivate default features in order to be able to use this on systems without
# access to an entropy souce via getrandom such as wasm32-unknown-unknown
rand = { version = "0.8", default-features = false }
```

For future you

**Comment by ewoolsey** [comment]
Thank you!

**Pull request merged**